content/reference-rippled.md

```diff
@@ -10955,6 +10957,182 @@ The response follows the [standard format](#response-formatting), with a success
 
 * Any of the [universal error types](#universal-errors).
 
+
+
+
+# Peer Protocol #
+
+Servers in the Ripple Consensus Ledger communicate to each other using the Ripple peer protocol, also known as RTXP. Peer servers connect via HTTPS and then _upgrade_ to RTXP. (For more information, see the [Overlay Network article in the `rippled` repository](https://github.com/ripple/rippled/blob/906ef761bab95f80b0a7e0cab3b4c594b226cf57/src/ripple/overlay/README.md#handshake).)
+
+## Configuring the Peer Protocol ##
+
+In order to participate in the Ripple Consensus Ledger, `rippled` servers connects to arbitrary peers using the peer protocol. (All such peers are treated as untrusted.)
```

Comment:
Partially correct. rippled supports "cluster peers" which are designated node public keys that have special privileges. For example, we trust that they verify signatures correctly. We operate such a cluster ourselves.